protocol/activity_center_persistence.go

```diff
@@ -697,6 +697,52 @@ func (db sqlitePersistence) GetActivityCenterNotificationsByID(ids []types.HexBy
 	return notifications, nil
 }
 
+func (db sqlitePersistence) GetActivityCenterNotificationsByTypeAuthorAndChatID(acType ActivityCenterType, author string, chatID string) (*ActivityCenterNotification, error) {
+	if len(author) == 0 {
+		return nil, nil
+	}
+	if len(chatID) == 0 {
+		return nil, nil
+	}
+	// nolint: gosec
+	// rows, err := db.db.Query(
+	query := `
+		SELECT
+			a.id,
+			a.timestamp,
+			a.notification_type,
+			a.chat_id,
+			a.community_id,
+			a.membership_status,
+			a.read,
+			a.accepted,
+			a.dismissed,
+			a.deleted,
+			a.message,
+			c.last_message,
+			a.reply_message,
+			a.contact_verification_status,
+			c.name,
+			a.author,
+			a.token_data,
+			a.updated_at,
+			a.installation_id
+		FROM activity_center_notifications a
+		LEFT JOIN chats c ON c.id = a.chat_id
```

Comment:
ouch, I see that not you started this, but can you please extract this mess to a constant?
Perhaps `protocol/persistence_metrics.go` can be an okay-ish example.

Or go with [Squirrel](https://github.com/Masterminds/squirrel), as here:
https://github.com/status-im/status-go/blob/cfcef92e64ad1c2e8c85cce6a668f9091c7d717f/transactions/pendingtxtracker_db.go#L43-L45

But I leave it at your decision if you want to refactor this now ofc.